### ISSUE TITLE
[question] Can this work on FreeBSD?

### ISSUE THREAD

**Issue by yurivict** [blueman-project/blueman#901]
How does it access bluetooth?

**Comment by cschramm** [comment]
I don't think so. blueman heavily relies on BlueZ, mostly on its DBus API. While the FreeBSD bluetooth service seems to provide some level of compatibility to BlueZ, it does not seem to provide that API.
